tests/test_data/vegrecipesofindia.com/vegrecipesofindia.json

```diff
@@ -1,80 +0,0 @@
-{
-  "author": "Dassana Amit",
-  "canonical_url": "https://www.vegrecipesofindia.com/sarson-ka-saag/",
```

Comment:
I notice many of the other recent ingredient-grouping pull requests retain the existing test and add a supporting one - is there a reason the existing test is removed in this PR?

Reply:
The original test recipe contained groupings already so just swapped it out so we'd maintain one with & without.

Reply:
Ok, seems reasonable - thanks!